ci/build_wheel_ucxx.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_dir="python"
+
+export RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+./ci/build_wheel.sh ucxx ${package_dir}
+
+RAPIDS_PY_WHEEL_NAME="ucxx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 ${package_dir}/final_dist
```

Comment:
Same comment as in build_wheel_distributed_ucxx.sh

Reply:
Still to-do

Reply:
Also done in https://github.com/rapidsai/ucxx/pull/167/commits/2a8495df31ad1c563a7eba0967f39783006e7cc6